src/pages/api/claim.ts

```diff
@@ -1,17 +1,13 @@
 import { NextApiRequest, NextApiResponse } from 'next'
-import merkle from 'src/constants/roots.json'
+import merkles from 'src/constants/roots.json'
 
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse
 ): Promise<void> {
-  const proof =
-    merkle[req.body.chainId.toString()].claims[req.body.address.toString()]
-  if (!proof) {
-    res.status(404).json({ error: 'proof not found' })
```

Comment:
dropped the 404?

Reply:
yeah, I just opted to always return something since it was simpler